config/locales/server.en.yml

```diff
@@ -878,6 +878,8 @@ en:
     download_remote_images_to_local: "Convert remote images to local images by downloading them; this prevents broken images."
     download_remote_images_threshold: "Minimum disk space necessary to download remote images locally (in percent)"
     download_remote_images_max_days_old: "Don't download remote images for posts that are more than n days old."
+    download_onebox_images_to_local: "Convert onebox images to local images by downloading them; this prevents broken images."
+    download_only_http_onebox_images: "Download onebox images only if it does not have SSL certificate."
```

Comment:
Not sure that site setting is really useful. It complicates the setup & code for no reasons.